Integrations/integration-Panorama.yml

```diff
@@ -91,17 +93,11 @@ script:
     # determine a vsys or a device-group
     VSYS = demisto.params().get('vsys')
     DEVICE_GROUP = demisto.params().get('device_group')
+    # configuration check
     if DEVICE_GROUP and VSYS:
-        return_error('Can not configure both vsys and Device Group. Set vsys for Firewall, set Device Group for Panorama')
-    if not DEVICE_GROUP and not VSYS:
-        return_error('Set a vsys for Firewall or Device Group for Panorama')
-    # setting security xpath relevant to FW or panorama managment
-    XPATH_SECURITY_RULES = None
-    if DEVICE_GROUP:
-        XPATH_SECURITY_RULES = "/config/devices/entry/device-group/entry[@name=\'" + DEVICE_GROUP + "\']/"
-    else:
-        XPATH_SECURITY_RULES = "/config/devices/entry/vsys/entry[@name=\'" + VSYS + "\']/rulebase/security/rules/entry"
-
+            return_error('Can not configure both vsys and Device Group. Set vsys for Firewall, set Device Group for Panorama')
```

Comment:
Firewall => firewall
Device Group => Device group/device group
Can not => Cannot